usecases/network_statistics_api.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from elasticsearch import exceptions
+
+from common.errors import ApiError
+from common.logging import get_logger
+from gateways.network_statistics_api_gateway import NetworkStatisticsApiGateway
+
+logger = get_logger()
+
+
+class NetworkStatisticsApi:
+    def __init__(
+        self,
+        network_statistics_api_gateway: Optional[NetworkStatisticsApiGateway] = None,
+    ) -> Optional[None]:
+        self.network_statistics_api_gateway = (
+            network_statistics_api_gateway or NetworkStatisticsApiGateway()
+        )
+
+    def get_basic_statistics(self) -> dict:
+        try:
+            return self.network_statistics_api_gateway.get_transaction_statistics()
+        except exceptions.ApiError as err:
+            logger.error(
+                "ElasticSearch request error",
```

Comment:
This message will eventually appear in alerts, right?

Maybe add some more details to it, like "ExplorerService failed to fetch statistics from ElasticSearch"

Reply:
Done: 654d151b516001b6086e273e148e2b2e8c3964e4